src/Snackbar/Snackbar.js

```diff
@@ -169,7 +170,7 @@ type Props = DefaultProps & {
    * @param {event} event The event that triggered the close request
    * @param {string} reason Can be:`"timeout"` (`autoHideDuration` expired) or: `"clickaway"`
    */
-  onRequestClose?: (event: ?SyntheticUIEvent, reason: string) => void,
+  onRequestClose?: (event: ?Event, reason: string) => void,
```

Comment:
Best cheatsheet for flow: https://www.saltycrane.com/blog/2016/06/flow-type-cheat-sheet/